src/Managed.sol

```diff
@@ -81,10 +89,36 @@ abstract contract Managed is UUPSUpgradeable {
         return _getAdmin();
     }
 
-    /// @notice Changes the admin of the contract.
+    /// @notice Returns the proposed address to change the admin to.
+    function proposedAdmin() public view returns (address) {
+        return _managedStorage().proposedAdmin;
+    }
+
+    /// @notice Proposes a change of the admin of the contract.
+    /// The proposed new admin must call `acceptAdmin` to finalize the change.
+    /// To cancel a proposal propose a different address, e.g. the zero address.
+    /// Can only be called by the current admin.
+    /// @param newAdmin The proposed admin address.
+    function proposeAdmin(address newAdmin) public onlyAdmin {
```

Comment:
```suggestion
    function proposeNewAdmin(address newAdmin) public onlyAdmin {
```
Maybe?
`proposeAdmin` and `proposedAdmin` are so simliar.

Reply:
Good idea, applied.